pykechain/models/widgets/widget.py

```diff
@@ -83,7 +83,7 @@ def title_visible(self) -> Optional[str]:
             try:
                 if self.widget_type == WidgetTypes.PROPERTYGRID:
                     return self._client.part(
-                        pk=self.meta.get(AssociatedObjectId.PART_INSTANCE_ID)
+                        pk=self.meta.get(AssociatedObjectId.PART_MODEL_ID)
```

Comment:
I think this would depend on the category of the form, if it's of a category INSTANCE, the default name should actually be that of the part instance.

Reply:
Done, added an if-else statement to check if the `self.meta.get(AssociatedObjectId.PART_INSTANCE_ID)` exists first